app/src/main/java/com/github/yeriomin/yalpstore/view/AppBadge.java

```diff
@@ -35,10 +35,12 @@ public void draw() {
         view.findViewById(R.id.progress).setVisibility(View.GONE);
         view.findViewById(R.id.app).setVisibility(View.VISIBLE);
 
-        ((TextView) view.findViewById(R.id.text1)).setText(app.getDisplayName());
+        ((TextView) view.findViewById(R.id.installed_app_name)).setText(app.getDisplayName());
+        ((TextView) view.findViewById(R.id.installed_app_pname)).setText(app.getPackageName());
+        ((TextView) view.findViewById(R.id.installed_app_version)).setText(app.getVersionName());
```

Comment:
It is not a good idea to change view ids. On different pages they contain different info. They are supposed to have generic names.